azkaban-common/src/main/java/azkaban/executor/FetchActiveFlowDao.java

```diff
@@ -196,7 +195,7 @@ static class FetchActiveExecutableFlows implements
             + " LEFT JOIN "
             + " executors et ON ex.executor_id = et.id"
             + " WHERE ex.status NOT IN ("
-            + getTerminatingStatusesString() + ")";
+            + FetchActiveFlowDao.getTerminatingStatusesString() + ")";
```

Comment:
hey !
 Can you please elaborate more on whats being changed here?

Reply:
They're from style guide lines :)
https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md#style-guides